src/gtk-2.0/gtkrc.hidpi

```diff
@@ -267,17 +267,16 @@ style "murrine-progressbar" = "murrine-thin" {
 }
 
 style "murrine-treeview-header" = "murrine-button" {
-	#bg[NORMAL] = @base_color
-	bg[NORMAL] = mix(0.12, @bg_color, @base_color)
-	bg[PRELIGHT] = shade (1.04, @base_color)
-	bg[SELECTED] = shade (0.7, @base_color)
-	bg[ACTIVE] = shade (0.95, @base_color)
+	bg[NORMAL] = @bg_color
```

Comment:
the only remaining thing now is to remove color styling in gtk2 theme (so it will match new nice style in gtk+3 theme)

Reply:
gtk+2 seems to be a little different. The buttons have a flat background, and if I remove those colors for the column headers, they still don't look as buttons. It seems that their appearance depends on the "listviewheaderstyle" options, which only accepts 3 values:
listviewheaderstyle = 0 # 0 = flat, 1 = glassy, 2 = raised

Reply:
By the way, I should also change the style for gtk+ 3.0. It's not used for the screenshots, but I have an ubuntu LTS where I can test it by eye

Reply:
in that case mb let's just set its colors to button_bg and button_fg accordingly to at least match gtk+3 theme by color

gtk< 3.18 is sorta deprecated, ie i am not removing it by neither doing any updates but if you are volunteering to update it feel free to do that